src/resources/js/events-admin.js

```diff
@@ -694,7 +694,7 @@ jQuery( document ).ready( function( $ ) {
 	//show state/province input based on first option in countries list, or based on user input of country
 	$( 'body' ).on( 'change', '#EventCountry', function () {
 		var $country        = $( this );
-			$container      = $country.parents( 'div.eventForm' ).eq( 0 ),
+		$container      = $country.parents( 'div.eventForm' ).eq( 0 ),
```

Comment:
Alignment is weird on this line.

Reply:
Yeah scoping is crappy too, now you point it out. Will fix.